docs/platform/howto/restrict-access.md

```diff
@@ -5,40 +5,42 @@ sidebar_label: Restrict access
 
 import ConsoleLabel from "@site/src/components/ConsoleIcons"
 
-By default, Aiven services are publicly accessible, but you can restrict access to your service to a single IP, an address block, or any combination of both.
+Restrict access to your Aiven-managed service to a single IP, an address block, or any combination of both.
 
-:::important
-IP filters apply to inbound traffic coming from specified IP addresses / network ranges to
-Aiven service ports.
+By default, Aiven services are publicly accessible. To restrict access to Aiven services,
+you can use the IP filtering capability, which allows you to filter traffic incoming to
+your services by specifying allowed IP addresses / network ranges.
+
+note:::
+If your service is within a VPC, the VPC configuration filters incoming traffic before the
+IP filter is applied.
 :::
 
+By default, the IP filter is set `0.0.0.0/0`, which allows all inbound connections. If you
+remove `0.0.0.0/0`, your service loses all connectivity, including private network addresses.
```

Comment:
... If you remove 0.0.0.0/0 without adding the networks or addresses used by the clients, no clients will be able to connect.

(The service does not loose "all connectivity" since many services can still do outbound connections, integrations etc; the filter only applies to client connections. The 0.0.0.0/0 also needs to be removed when adding the actual correct client networks in the ACL, otherwise all addresses are still allowed to connect.